src/client/Client.js

```diff
@@ -357,7 +357,18 @@ class Client extends BaseClient {
       .get()
       .then(app => new ClientApplication(this, app));
   }
-
+  /**
+   * Obtains the widget from a guild, available for all guilds
+   * @param {GuildResolvable} guild The guild to fetch the widget for
+   * @returns {Promise<Object>}
+   */
+  fetchWidget(guild) {
+    const id = this.guilds.resolveID(guild);
+    if (!id) throw new TypeError('INVALID_TYPE', 'guild', 'GuildResolvable');
+    return this.api
+      .guilds(id)
```

Comment:
Apparently ESLint wants you to do this, other than that, LGTM once CI turns green:
```suggestion
    return this.api.guilds(id)
```